server/webserver.go

```diff
@@ -451,9 +451,34 @@ func (s *LivepeerServer) StartWebserver() {
 	})
 
 	http.HandleFunc("/status", func(w http.ResponseWriter, r *http.Request) {
-		w.Write([]byte(fmt.Sprintf("VideoDB: %v", s.LivepeerNode.VideoDB)))
-		w.Write([]byte(fmt.Sprintf("\n\nVideoNetwork: %v", s.LivepeerNode.VideoNetwork)))
-		w.Write([]byte(fmt.Sprintf("\n\nmediaserver sub timer: %v", s.hlsSubTimer)))
+		nid := r.FormValue("nodeID")
+		if nid == "" || string(s.LivepeerNode.Identity) == nid {
```

Comment:
I think this should be in a separate endpoint because it's really a different use case: getting your own debug info is different than getting a node's status (including your own status).